src/phoenix/server/api/queries.py

```diff
@@ -531,12 +528,13 @@ async def node(self, id: GlobalID, info: Info[Context, None]) -> Node:
                     raise NotFound(f"Unknown user: {id}")
             return to_gql_project_session(project_session)
         elif type_name == Prompt.__name__:
-            return Prompt(
-                id_attr=node_id,
-                name="Prompt " + str(node_id),
-                description="description",
-                created_at=datetime.now(),
-            )
+            async with info.context.db() as session:
+                if orm_prompt := await session.scalar(
+                    select(models.Prompt).where(models.Prompt.id == node_id)
+                ):
+                    return to_gql_prompt_from_orm(orm_prompt)
+                else:
+                    raise NotFound(f"Unknown prompt: {id}")
```

Comment:
This was producing inconsistent prompt payloads if you fetched via `prompts` query vs `nodes(globalId) { ... on Prompt {...} }`

Particularly the name formatting differed and the description was hardcoded.